### ISSUE TITLE
fixed palette mode

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#10]
users should be able to turn off the game's palette variation and use only a specific palette of their choosing in place of any given place the game uses a variable palette (or its own choice of palette)

**Comment by HeladoDeBrownie** [comment]
http://gameaccessibilityguidelines.com/provide-a-choice-of-text-colour-lowhigh-contrast-choice-as-a-minimum/
